### PR TITLE
Adding static page basic functionality

### PR DESCRIPTION
**Pull request by bmw95**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <title>Checkit</title>
+  <title>Checkit - App</title>
```
Comment (lasley) [inline app/views/layouts/application.html.erb]: http://stackoverflow.com/questions/21332504/dynamic-titles-for-each-page

File: app/views/layouts/static_pages.html.erb [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Checkit | <%= yield(:page_title) %></title>
```
Comment (lasley) [inline app/views/layouts/static_pages.html.erb]: After I made the dynamic title comment, I saw this. Why the two title implementations?

File: spec/controllers/static_pages_controller_spec.rb [hunk 3]
```diff
@@ -1,8 +1,22 @@
 require 'rails_helper'
+
 RSpec.describe StaticPagesController, type: :controller do
```
Comment (lasley) [inline spec/controllers/static_pages_controller_spec.rb]: Woot great job on the basic tests. Everyone always forgets to see if the danged page loads!

Comment (bmw95) [inline spec/controllers/static_pages_controller_spec.rb]: Haha thanks man :)

File: app/views/layouts/static_pages.html.erb (hunk 2 above)
Comment (bmw95) [inline app/views/layouts/static_pages.html.erb]: Well for the static pages template layout you have different pages and such, so I want to have that dynamic title, but when you're in the app, it's just the app, so I decided to have a static title. However I might add dynamic titles to the app layout in the future though. Haven't quite decided yet.

Comment (lasley) [inline app/views/layouts/static_pages.html.erb]: Cool, just making sure it was a design decision vs an oversight :+1:

**Pull request merged**